react/Viewer/Readme.md

```diff
@@ -144,6 +169,9 @@ const PanelContent = ({ file }) => {
                       showPanel: ({ file }) => file.class === "pdf" || file.class === "audio",
                       PanelContent
                     }}
+                    mobileActionsProps={{
+                      MobileActionsContent
+                    }}
```

Comment:
I don't think we should call that mobileActions. We should be more agnostic in our Viewer and having props like "toolbar, footer" and so one

Reply:
I'm saying this because I've some use cases where our partners wants to be able to customize our "blocks".  And one of them was the footer.